app/src/main/res/values/strings.xml

```diff
@@ -261,7 +266,8 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="quest_busStopShelter_name_title">"Does the bus stop \"%s\" have a shelter?"</string>
     <string name="quest_bin_name_title">"Does the bus stop \"%s\" have a waste bin?"</string>
     <string name="quest_passengerInformation_name_title">"Does the bus stop \"%s\" have an electronic information display?"</string>
-    <string name="quest_busStopLit_name_title">"Is the bus stop \"%s\" well lit?"</string>
+    <string name="quest_busStopLit_name_title">"Is there lighting at the \"%s\" Bus stop?"</string>
+    <string name="quest_busStopLit_question2">"Are there one or more street lamps within 10 feet of this Bus stop?"</string>
```

Comment:
I would ask a vaguer question, because the tag itself is also vaguer: is this bus stop lit?